### ISSUE TITLE
Issue while trying to update metadata using "node utils/update_info.js"

### ISSUE THREAD

**Issue by atamproject** [HashLips/hashlips_art_engine#1179]
Really struggling with the update of my metadata while trying to update my CID into the config.js file. When I type : **node utils/update_info.js**
Command prompt goes : 
**throw err;
    ^

Error: ENOENT: no such file or directory, open 'C:\Users\XXX\Downloads\hashlips_art_engine-main/build/json/_metadata.json'
    at Object.openSync (node:fs:585:3)
    at Object.readFileSync (node:fs:453:35)
    at Object.<anonymous> (C:\Users\XXX\Downloads\hashlips_art_engine-main\utils\update_info.js:14:18)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  errno: -4058,
  syscall: 'open',
  code: 'ENOENT',
  path: 'C:\\Users\\XXX\\Downloads\\hashlips_art_engine-main/build/json/_metadata.json'**
  
  Hoping to find some help here !
  Thanks for reading

**Comment by bolshoytoster** [comment]
@atamproject you're missing the `build/json/_metadata.json` file. Have you deleted it or did a collection not finish generating?

**Comment by atamproject** [comment]
I didn't get any kind of file with this name while generating but when the generating session was done it said everything was done. Idk how to solve my problem. :(

**Comment by bolshoytoster** [comment]
@atamproject when it was generating did it say `You need more layers or elements to grow your edition to ... artworks!`?

Either way, try regenerating and if that doesn't work try using [my fork](https://github.com/bolshoytoster/hashlips_art_engine).

**Comment by atamproject** [comment]
Yes it did. Is that the reason why this issue occurs ? Cauz I calculated befaure like it was 11^4 so my collection would've been 14641 pieces but it stopped at about 11000 or so. Don't know why tho, I'm probably missing something. Anyway thanks ! I'll regenerate everything tomorrow :)

**Comment by bolshoytoster** [comment]
@atamproject that is why.

HashLips generates by creating a random DNA and checking if it's unique. If it generates too many duplicates it will quit.

The main repo doesn't generate `_metadata.json` if it generates too many duplicates, but my fork generates it regardless.

You could generate `_metadata.json` without completely regenerating by running [my `utils/update_metadata.js`](https://raw.githubusercontent.com/bolshoytoster/hashlips_art_engine/main/utils/generate_metadata.js).

**Comment by SwagCord13** [comment]
> @atamproject that is why.
> 
> HashLips generates by creating a random DNA and checking if it's unique. If it generates too many duplicates it will quit.
> 
> The main repo doesn't generate `_metadata.json` if it generates too many duplicates, but my fork generates it regardless.
> 
> You could generate `_metadata.json` without completely regenerating by running [my `utils/update_metadata.js`](https://raw.githubusercontent.com/bolshoytoster/hashlips_art_engine/main/utils/generate_metadata.js).

I used a different software to generate metadata and images. I dropped them into the code and made all the config.js all good and all folders all good. However i am having same problem. Like i have all the images and metadata just need to update the metadata

**Comment by bolshoytoster** [comment]
@SwagCord13 so you just want to update the metadata? Is there an issue with just running `node utild/update_metadata.js`?
